apps/web/src/composables/iopay.ts

```diff
@@ -28,6 +28,10 @@ export default function useIoPay() {
     }
   }
 
+  const signIoTexMessage = async (message: string) => {
```

Comment:
Change to `signIoPayMessage` for consistency.

Reply:
Resolved.